### PR TITLE
Chore: Use new Tag instead of nnsdapp HeadingTag

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Be consistent with tags.

In this PR, remove the HeadingTag and instead use the Tag with size "large"

# Changes

* In NnsNeuroPageHeading and SnsNeuronPageHeading use a Tag with size "large" instead of HeadingTag.

# Tests

Tests still passing. Test ids were preserved.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
UI has changed slightly, but not that it matters, we want the new tags.

Screenshot 2023-10-10 at 08 14 06
Screenshot 2023-10-10 at 08 14 12

**Comment by lmuntaner** [comment]
@dskloetd @peterpeterparker please review

File: frontend/src/lib/components/neuron-detail/NnsNeuronPageHeading.svelte
Comment (dskloetd) [inline frontend/src/lib/components/neuron-detail/NnsNeuronPageHeading.svelte:1]: What is the purpose of `HeadigTag` if not to be used in a heading?

Comment (lmuntaner) [reply frontend/src/lib/components/neuron-detail/NnsNeuronPageHeading.svelte:1]: I'm removing the `HeadingTag`, I'm not sure I understand the question.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**